cmd/server.go

```diff
@@ -254,6 +255,10 @@ var boolFlags = map[string]boolFlag{
 		description:  "Disable \"atlantis apply\" command so a specific project/workspace/directory has to be specified for applies.",
 		defaultValue: false,
 	},
+	DisableAutoPlanFlag: {
+		description:  "Disable atlantis auto planning feature",
+		defaultValue: false,
+	},
```

Comment:
Can you add this flag to server_test.go so it gets tested.

Reply:
Sure thing !